scripts/linters/warranted_angular_security_bypasses.py

```diff
@@ -26,5 +26,7 @@
 #  have a longer review time. This is because the reviewers will have to ensure
 #  that the introduction of bypassSecurityTrust was the only option.
 
-EXCLUDED_BYPASS_SECURITY_TRUST_FILES = ()
+EXCLUDED_BYPASS_SECURITY_TRUST_FILES = (
+    'core/templates/services/svg-sanitizer.service.spec.ts',
+    'core/templates/services/svg-sanitizer.service.ts')
```

Comment:
Is this a temporary or long-term exclusion? We should avoid making this permanent--can you add justification here in code explaining why these need to be excluded, and file an issue for removing them from the excluded list & adding a TODO for that here?

Reply:
Unfortunately, this is a permanent inclusion in the EXCLUDED_BYPASS_SECURITY_TRUST_FILES. There is nothing we can do to make this inclusion temporary.

Reply:
But I have added a comment explaining why the files are present in the list.

Reply:
Got it--thanks for the context.

I think for long-term security bypasses we should keep them as isolated as possible. Could we maybe introduce a Base64ImageTrustedSanitizer class that basically just does this bypass & nothing else? We should also clarify in that file that it shouldn't really grow. This lets us avoid accidentally doing other security bypasses in the future since the svg-sanitizer class may have additional functionality added in the future.

Also @seanlip any thoughts on the naming for the new proposed sanitizer? This is a bit of a new pattern that we're establishing for classes with special long-term security bypasses.

Reply:
Another possible solution to this is that we count the number of times the phrase "bypass" occurs in the file and match it with a value in the security_bypasses.py file. Also, all these files listed here should have either @BenHenning or @seanlip as at least one of the code owners.

Reply:
Hmm, I'm not opposed to the count check but it doesn't feel like it's solving quite the same thing (e.g. making sure that unrelated changes don't improperly use bypass). It stops one case (increasing/decreasing the number of calls), but doesn't handle other scenarios like changing code in a way that's not correct. Codeowners helps, but one benefit of isolating the bypass call is that we don't need to review unrelated changes that aren't tied to security.

So that I can better understand your perspective: what are your concerns with having a separate file for the purpose of isolating the call?

Reply:
Actually, We can move this out into a different file. My only concern was to keep the code that checks and the code that bypasses into the same file. Which is why I was not so keen on this. The checks are equally important if not more. They should be treated with the same level of attention we want to achieve with bypassSecurity function because these checks are the ones that ultimately decide what is safe and what is not.

Reply:
As discussed in chat, given that we don't expect the sanitizer to change much I'm fine with keeping things as-is. However, please do make the suggested changes for codeowners (e.g. please make sure the codeowners of the security check bypass list is the same as all of the files on that list so that we can audit them & ensure they don't change in unexpected ways).

Reply:
Done!